narwhals/utils.py

```diff
@@ -1062,6 +1062,6 @@ def generate_repr(header: str, native_repr: str) -> str:
 
 
 def check_column_exists(columns: list[str], subset: list[str] | None) -> None:
-    if subset is not None and any(x not in columns for x in subset):
-        msg = f"Column(s) {subset} not found in {columns}"
+    if subset is not None and (missing := set(subset).difference(columns)):
+        msg = f"Column(s) {', '.join(missing)} not found in {columns}"
```

Comment:
shall we just print `{missing}`? I think sets render quite nicely

Reply:
Sure.. (It was printing the brackets as well)

Reply:
And the errors are making me crazy because I don't have them locally..

Reply:
Oh, sometimes the order of the columns is d, f and sometimes f, d. Sometimes it fails.

Reply:
Ah I see sorry - does it work to just call `sorted(missing)`?

Reply:
yes, thank you.